xml_inscriptions/transcribed/MD.Balt.WAG.L.48.336.xml

```diff
@@ -0,0 +1,182 @@
+                            <date evidence="#language" notBefore="-0001" notAfter="0100">1st c.
+                                CE</date>
+                            <placeName ref="unknown">Roman Empire</placeName>
+                        </origin>
+                        <provenance type="transferred" subtype="purchased">
+                            <p>
+                                <date when="1902">1902</date> - <persName role="buyer"><name key="Henry Walters">Henry Walters</name></persName>, <placeName>Baltimore,</placeName> from <orgName><persName role="collector"><name key="Don Marcello Massarenti">Don Marcello Massarenti</name></persName> Collection</orgName>, <placeName>Rome</placeName>.
+                            </p>
+                            
+                        </provenance>
+                        <provenance type="transferred" subtype="donated">
+                            <p><date when="1931">1931</date> - <orgName>Walters Art Museum</orgName>, by bequest.
+                            </p>
+                            
+                        </provenance>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl> 
+                        <ptr type="rest-of-bibl" target="#vanEsbroeck"/>
+                        <biblScope type="vol">2</biblScope>
+                        <biblScope type="page">59</biblScope>
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2020-10-23" who="Itamar Levin">metadata and translation</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url="MD.Balt.WAG.L.48.336.jpg"/>
+        </surface>
+    </facsimile>
+
+    <text>
+        <body>
+            <div type="edition">
+            <ab>
+                <lb n="1"/><persName><name type="nomen" key="Clonius">Cloni</name> 
```

Comment:
Note the changes here; this is almost certainly an imperial slave (perhaps the same Clonius Caesaris, who is later freed and continues to work as an imperial freedman (T. Flavius Aug. l. Clonius, whom you mention in your cf. in the notes).